Dockerfile

```diff
@@ -24,17 +14,33 @@ RUN wget -O /root/.torch/models/vgg16_bn-6c64b313.pth https://download.pytorch.o
 
 RUN wget -O /root/.torch/models/resnet34-333f7ec4.pth https://download.pytorch.org/models/resnet34-333f7ec4.pth
 
-RUN wget -O /data/models/ColorizeArtistic_gen.pth https://www.dropbox.com/s/zkehq1uwahhbc2o/ColorizeArtistic_gen.pth?dl=0 
+#Resnet 101 is used by stable and resnet 34 by artistic
+
+RUN wget -O /root/.torch/models/resnet101-5d3b4d8f.pth https://download.pytorch.org/models/resnet101-5d3b4d8f.pth
 
 ADD . /data/
 
+#incase models/ColorizeArtistic_gen.pth is already available
```

Comment:
I'll write something for that :

COPY *.pth /data/models/

to avoid missing files error

then 
RUN [[ ! -f models/ColorizeArtistic_gen.pth ]] && wget https://data.deepai.org/deoldify/ColorizeArtistic_gen.pth -O ./models/ColorizeArtistic_gen.pth

something like that to manage cache like case.